exercises/practice/parallel-letter-frequency/.meta/config.json

```diff
@@ -1,6 +1,59 @@
 {
   "blurb": "Count the frequency of letters in texts using parallel computation.",
   "authors": [],
```

Comment:
another empty one, added in https://github.com/exercism/elixir/commit/1b64e528d2c18f9ec392b239f1107bfa4b53434c